vispy/io/wavefront.py

```diff
@@ -303,10 +305,14 @@ def writeMesh(self, vertices, faces, normals, values, name=''):
         if faces is None:
             faces = np.arange(len(vertices))
 
-        # Reshape faces
-        Nfaces = faces.size // 3
-        faces = faces.reshape((Nfaces, 3))
-
+        try:
+            # Reshape faces
+            Nfaces = faces.size // 3
+            faces = faces.reshape((Nfaces, 3))
```

Comment:
hmm... but couldn't you have a non-triangular file that just happened to be divisibly by 3, thus making this "check" not correct?